hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/KafkaOffsetGen.java

```diff
@@ -189,6 +189,11 @@ public static long totalNewMessages(OffsetRange[] ranges) {
             .defaultValue(KafkaResetOffsetStrategies.LATEST)
             .withDocumentation("Kafka consumer strategy for reading data.");
 
+    public static final ConfigProperty<String> JSON_KAFKA_PROCESSOR_CLASS_OPT = ConfigProperty
+        .key("hoodie.deltastreamer.source.kafka.json.processor.class")
+        .noDefaultValue()
+        .withDocumentation("Json kafka source pre-processor class name, used to do pre-process on kafka data.");
```

Comment:
lets not use "pre-process" as it might be confusing. we can just say post process data after consuming from source and before giving it to deltastreamer.

Reply:
> lets not use "pre-process" as it might be confusing. we can just say post process data after consuming from source and before giving it to deltastreamer.

done